source/common/logger/ur_level.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
+
+namespace level {
+
+enum class Level {
+    DEBUG,
```

Comment:
Is such uppercase enforced by your coding style ?

Reply:
Yes, other enums in this project are uppercase. See Naming Convention here: https://github.com/oneapi-src/unified-runtime/blob/7bc754a218eee13d082bbcfcd8f221a541cbf211/scripts/core/INTRO.rst